### PR TITLE
Add Carthage Support for other platforms

### LINKED ISSUE #25: Use carthage to install
I try Carthage to install this module using in cartfile:

`github "kylef/JSONWebToken.swift"`

but on `carthage update --platform iOS`

I receive the error:

`Dependency "JSONWebToken.swift" has no shared framework schemes for any of the platforms: iOS`

Can you help me?
Thanks.

### PR DESCRIPTION
**Pull request by ikesyo**
Fixes #25.

### CODE REVIEW

File: Cartfile.resolved [hunk 1]
```diff
@@ -1 +1,2 @@
-github "krzyzanowskim/CryptoSwift" "0.2.2"
+github "krzyzanowskim/CryptoSwift" "0.2.3"
+github "jspahrsummers/xcconfigs" "0.9"
```
Comment (kylef) [inline Cartfile.resolved]: Can you please explain why you've added this dependency?

Comment (ikesyo) [inline Cartfile.resolved]: To remove boilerplate build settings in xcodeproj. The build targets are now based on some `.xcconfig` files.

Comment (kylef) [inline Cartfile.resolved]: I'd rather we kept it as it is. This adds additional complexity and multiple different files (25 of them) to track project settings. Would you be able to remove this from the pull request?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,66 @@
...
+
+# fastlane
+#
+# It is recommended to not store the screenshots in the git repo. Instead, use fastlane to re-generate the 
+# screenshots whenever they are needed.
+# For more information about the recommended setup visit:
+# https://github.com/fastlane/fastlane/blob/master/docs/Gitignore.md
+
+fastlane/report.xml
+fastlane/screenshots
```
Comment (kylef) [inline .gitignore]: We don't even use fastlane and half of the stuff in here. Could you remove these additions?

Comment (ikesyo) [inline .gitignore]: Hm, this was just the contents of https://github.com/github/gitignore/blob/a392c48ccf4136ef75acb05b7ead4e9c75235c19/Swift.gitignore. Anyway I'll update this as per your comment.

**Comment by ikesyo** [comment]
@kylef I've updated the PR as per your requests.

**Comment by kylef** [comment]
Thanks!

Merged via 6b6ca01.

**Pull request closed without merging**